src/lib/ndpi_main.c

```diff
@@ -1963,6 +1964,7 @@ struct ndpi_detection_module_struct *ndpi_init_detection_module(ndpi_init_prefs
 
 #ifdef NDPI_ENABLE_DEBUG_MESSAGES
   set_ndpi_debug_function(ndpi_str, (ndpi_debug_function_ptr) ndpi_debug_printf);
+  NDPI_BITMASK_RESET(ndpi_str->debug_bitmask);
```

Comment:
You've created the fancy function `ndpi_set_debug_bitmask` but not using it here. Why?

Reply:
This is the initialization, i.e. the default value.
I think, it is better to have a proper value in ndpi_str->debug_bitmask even if the user doesn't call ndpi_set_debug_bitmask()